### PR TITLE
Add new process collection in core agent flag

### PR DESCRIPTION
**Pull request by atrittman**
### What does this PR do? What is the motivation?
Live Processes is releasing a flag to reduce the footprint of the agent. Update the Live Processes documentation to include instructions for Helm and Operator.

### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/infrastructure/process/_index.md [hunk 1]
```diff
@@ -314,6 +314,35 @@ agents:
 {{% /tab %}}
 {{< /tabs >}}
 
+### Optimize footprint for process collection
+By default, the Datadog Agent has a separate Process Agent for process collection with a separate Process Agent. To reduce the Agent's footprint, you can have the consolidate container and process collection to the core Agent if you're running a Linux environment.
```
Comment (robertjli) [inline content/en/infrastructure/process/_index.md]:
```suggestion
By default, the Datadog Agent has a separate Process Agent for container and process collection. To reduce the Agent's footprint, you can consolidate container and process collection to the core Agent if you're running a Linux environment.
```

File: content/en/infrastructure/process/_index.md [hunk 2]
```diff
@@ -314,6 +314,35 @@ agents:
...
+  name: datadog
+spec:
+  override:
+    nodeAgent:
+      env:
+        - name: DD_PROCESS_CONFIG_RUN_IN_CORE_AGENT_ENABLED
+          value: "true"
+```
+{{% /tab %}}
+{{< /tabs >}}
```
Comment (robertjli) [inline content/en/infrastructure/process/_index.md:345]: Need a tab for agent configuration as well

**Review by robertjli — COMMENTED** [review]
What do you think about putting this section above "Process arguments scrubbing"?

File: content/en/infrastructure/process/_index.md (hunk 1 above)
Comment (atrittman) [reply content/en/infrastructure/process/_index.md]: Done

File: content/en/infrastructure/process/_index.md (hunk 2 above)
Comment (atrittman) [reply content/en/infrastructure/process/_index.md]: Done

File: content/en/infrastructure/process/_index.md [hunk 3]
```diff
@@ -232,6 +232,47 @@ I/O and open files stats can be collected by the Datadog system-probe, which run
...
+  override:
+    nodeAgent:
+      env:
+        - name: DD_PROCESS_CONFIG_RUN_IN_CORE_AGENT_ENABLED
+          value: "true"
+```
+{{% /tab %}}
+
+{{% tab "Linux Hosted" %}}
+If you are running the Agent outside of containers on Linux, add the `run_in_core_agent` flag in your `datadog-agent.yaml` file:
```
Comment (robertjli) [inline content/en/infrastructure/process/_index.md:265]:
```suggestion
If you are running the Agent outside of containers on Linux, add the `run_in_core_agent` flag in your `datadog.yaml` file:
```

Comment (atrittman) [reply content/en/infrastructure/process/_index.md]: Done

**Pull request closed without merging**